src/App.css

```diff
@@ -12,7 +12,7 @@
   padding: 24px 110px;
   max-width: 1184px;
   margin: auto;
-  overflow-x: hidden;
+  /*overflow-x: hidden;*/
```

Comment:
Can you remove this commented out line of CSS?

Reply:
done